ee/query-service/dao/sqlite/auth.go

```diff
@@ -67,15 +125,15 @@ func (m *modelDao) PrecheckLogin(ctx context.Context, email, sourceUrl string) (
 	// assume user is valid unless proven otherwise
 	resp := &model.PrecheckResponse{IsUser: true, CanSelfRegister: false}
 
-	// check if email is a valid user
-	userPayload, baseApiErr := m.GetUserByEmail(ctx, email)
-	if baseApiErr != nil {
-		return resp, baseApiErr
-	}
+	// // check if email is a valid user
+	// userPayload, baseApiErr := m.GetUserByEmail(ctx, email)
+	// if baseApiErr != nil {
+	// 	return resp, baseApiErr
+	// }
 
-	if userPayload == nil {
-		resp.IsUser = false
-	}
+	// if userPayload == nil {
+	// 	resp.IsUser = false
+	// }
```

Comment:
Desription says

> don't send isUser: false field when the user doesn't exist. Rather keep it true for SSO users

You are removing it entirely. Did you mean to do it only for SSO users? Otherwise, This will make the non-SSO users' experience confusing.  With this check removed, users will no longer see the _get invite link from admin_ message.

Reply:
right. It should not be removed entirely. `IsUser: true` should be set exceptionally when `SSO` feature is enabled. Sending a fix.

Reply:
I have tested the fix in the new commit for non-SSO admin register, non-SSO invite register and SSO users